### ISSUE TITLE
Calling `render()` results in error

### ISSUE THREAD

**Issue by simonsays1980** [Jjschwartz/NetworkAttackSimulator#27]
**Describe the bug**
Calling the `render()`-method of the NASim environment results in an error:
```
    259     self._renderer.render_readable(obs)
    260 else:
    261     print(
    262         "Please choose correct render mode from :"
--> 263         f"{self.rendering_modes}"
    264     )

AttributeError: 'NASimGymEnv' object has no attribute 'rendering_modes'
```

**To Reproduce**
Run the following code:
```
import gym
env = gym.make("nasim:Tiny-v0")
env.render()
```

**Expected behavior**
I expected either an environment rendering or to receive an error message that helps me to modify my function call. 

**Desktop (please complete the following information):**
 - OS: Fedora Linux 35

**Additional context**
The error is minor, as using the `mode="readable"` results in a clean rendering.

**Comment by Jjschwartz** [comment]
Hey, thanks for raising the issues. They were all things I'd missed and it's to get them fixed. They should hopefully all be fixed in the latest version `v0.9.1`.

Thanks for the contribution.

**Comment by simonsays1980** [comment]
> Hey, thanks for raising the issues. They were all things I'd missed and it's to get them fixed. They should hopefully all be fixed in the latest version `v0.9.1`.
> 
> Thanks for the contribution.

Awesome Jonathan! Thanks for the quick fixes!
